composer.json

```diff
@@ -136,13 +141,17 @@
     "scripts-descriptions": {
         "drupal:install": "Installs Starshot.",
         "drupal:install-dev": "Installs Starshot, with additional modules and configuration tweaks for development.",
+        "drupal:rebuild": "Rebuilds the codebase and reinstalls Starshot from scratch.",
         "drupal:run-server": "Runs Starshot using the PHP webserver and opens it in the default browser."
     },
     "scripts-aliases": {
         "drupal:install": [
             "si",
             "sin"
         ],
+        "drupal:rebuild": [
+            "rebuild"
```

Comment:
I'd maybe remove this alias; I don't think we necessarily want non-developers to be invoking it, and I don't think the removal of the `drupal:` prefix adds a ton of value, TBH.